.gitignore

```diff
@@ -0,0 +1,3 @@
+# dev
```

Comment:
i think we can remove this file from the pr!

Reply:
oh - those are local development files I don't want added to GitHub, that Intellij and I generated, and we wouldn't want to be committed from anyone else's local dev env either.

Reply:
I did change the description to be more clear